### PR TITLE
Add a license

### PR DESCRIPTION
**Pull request by simmel**
Now that it's properly released 🎉 you need to pick a license for it.

See https://choosealicense.com/ for a "pick your license adventure" ; )

I'm OK with anything but I prefer a permissive one like [MIT](https://choosealicense.com/licenses/mit/).

What do you think?

### CODE REVIEW

**Comment by ryarnyah** [comment]
I prefer an Apache-2.0 but MIT might be ok.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (ryarnyah) [inline LICENSE]: You need to change that when adding license

**Comment by simmel** [comment]
I was as confused as you. From what I read it was supposed to look like 
that(!)

But from the horses mouth: https://www.apache.org/licenses/LICENSE-2.0

On Thu, 2021-06-10 at 08:18:51 -0700, ryarnyah wrote:
***@***.*** commented on this pull request.

**Comment by ryarnyah** [comment]
🍾 LGTM. Can you rebase your PR and resolve conflict?

**Pull request merged**